pkg/api/v1alpha1/cloudstackmachineconfig_webhook.go

```diff
@@ -61,11 +64,18 @@ func (r *CloudStackMachineConfig) ValidateUpdate(old runtime.Object) error {
 
 	var allErrs field.ErrorList
 	allErrs = append(allErrs, validateImmutableFieldsCloudStackMachineConfig(r, oldCloudStackMachineConfig)...)
-	if len(allErrs) == 0 {
-		return nil
+
+	if err, fieldName, fieldValue := r.Spec.DiskOffering.Validate(); err != nil {
```

Comment:
same here, no need to have the fieldName and value as return

Reply:
discussed with Jia offline and reached an agreement that field name and value are needed for validateUpdate in webhook call.

In validateCreate, a simple bad request can be returned, but for validteUpdate, field name has to be specified in exception.